core/includes/theme.inc

```diff
@@ -591,7 +591,7 @@ function _theme_process_registry(&$cache, $name, $type, $theme, $path) {
 
       // Allow variable preprocessors for all theme hooks, whether the hook is
       // implemented as a template or as a function.
-      // Check for existing variable preprocessors. Ensure arrayness.
+      // Check for existing variable preprocessors. Ensure arrays.
```

Comment:
Both sentences do not seem right. I would rather remote that, as *Ensure arrays* just describes something that is already clear from the code. A variable is changed to an array in other cases, but no comment says that.

Reply:
Agreed.